### ISSUE TITLE
Warning message in loadFromQIIME2

### ISSUE THREAD

**Issue by FelixErnst** [microbiome/mia#51]
Hi @yiluheihei @tvborm 

`loadFromQIIME2` throughs a few warnings. Can you have a look?

https://felixernst.github.io/mia/reference/loadFromQIIME2.html

Thanks!

**Comment by TuomasBorman** [comment]
Those errors occur when TSE is created and phylogenetic tree is added there. Creating that phylogenetic tree does not give any errors. Those same warning came before yeasterday's modifications, but I did not pay attentiont to them that much.

For me, it is quite hard to say what is wrong. I tried to find the problem, but I could not. Those same 4 warnings came also with Chandler's data, so it is probable that the problem is not with data (duplicate data points etc...)

However, same .read_qza function is also used for fetching other data, and there is no problems

**Comment by yiluheihei** [comment]
I think the reason for this warning is that the node label of the phylogenetic tree is not unique and some nodes have no label.

**Comment by FelixErnst** [comment]
The question remains, why it is thrown 4 times? Is this in `mia` or in `TSE`? If it is the first, this has to be fixed once, with a warning, but not 4.

**Comment by FelixErnst** [comment]
I think it might be in `TSE`

@fionarhuang I saw the same warnings pop up in the manuscript on page 16. I think the warning is correct, but I am not so sure, about it appearing 4 times. That seems excessive.

**Comment by yiluheihei** [comment]
I also think it is in `TSE`, since there is no warning while I import QIIME2 results as `phyloseq`.

**Comment by FelixErnst** [comment]
> I also think it is in `TSE`, since there is no warning while I import QIIME2 results as `phyloseq`.

But thats a different code base, isn't it?

**Comment by yiluheihei** [comment]
No, the core (base) code I used to create `phyloseq` object is the same with it in `loadFromQIIME2`.

**Comment by TuomasBorman** [comment]
I agree that the problem is in TSE, because I ran the code one row at the time, and there were no errors until the function call 

```
TreeSummarizedExperiment(
        assays = S4Vectors::SimpleList(counts = feature_tab),
        rowData = taxa_tab,
        colData = sample_meta,
        rowTree = tree,
        referenceSeq = refseq
    )
```

and then those 4 warning came

**Comment by fionarhuang** [comment]
Hi all, 

The warning was from `convertNode` that was called several times in the constructor of `TreeSummarizedExperiment`. I have fixed the warning by checking only nodes that are used in the link data. Before,  all nodes of the tree are checked to see whether they have duplicated or missing labels. 

Overall, the warning should disappear in version `1.99.11` when testing on the example data provided in this issue.  Hope it works with the new version.

**Comment by FelixErnst** [comment]
Fixed! https://felixernst.github.io/mia/reference/loadFromQIIME2.html

Thanks!
